app/controllers/moderations_controller.rb

```diff
@@ -11,7 +11,7 @@ def index
       order("hotness_score DESC").limit(100)
     @articles = @articles.cached_tagged_with(params[:tag]) if params[:tag].present?
 
-    @rating_votes = RatingVote.where(article: @articles.pluck(:id), user: current_user)
+    @rating_votes = RatingVote.where(article: @articles, user: current_user)
```

Comment:
Same here, Rails will transform this in a single select statement, `@articles` are already going to be used by the view to render the page, there's no need to extract ids pre-emptively